lib/tasks/db_maintenance.rake

```diff
@@ -251,6 +250,47 @@ namespace :cartodb do
       }, threads, thread_sleep, database_host)
     end
 
+    desc 'Install/upgrade CARTODB SQL functions - only organization owners for Bloomberg merge code release'
```

Comment:
Let's not make this bloomberg specific, carto has mentioned they are also interested in this functionality. The real purpose of this is to only apply extensions once per database, something like `load_functions_for_all_dbs`.

Reply:
This function though is primarily to block the rebuilding of quotas.  I left the extension logic untouched in both rake tasks.  Is that also carto's interest?  Or are they only looking to take an upstream PR for the extension?